package.json

```diff
@@ -23,9 +23,10 @@
     "server": "pipenv run python network-api/manage.py runserver",
     "start": "npm i && npm run build-uncompressed && run-p server watch:**",
     "snyk": "snyk test --file=package.json",
+    "test:procfile": "node test/test-procfile.js",
     "test:eslint": "eslint --config ./.eslintrc.yaml \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js",
     "test:scss": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" --syntax scss",
-    "test": "npm run build && run-s test:**",
+    "test": "run-s test:** build",
```

Comment:
I don't understand why we ran the build before we ran the linting, so this line change ensures we never even build if linting fails.